README.md

```diff
@@ -1,21 +1,41 @@
-# frosty
+# `frosty`
+>yet another gamepad library for java
+
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/a141c62a0d074580a10974a5ece29348)](https://www.codacy.com/app/delta-leonis/frosty?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=delta-leonis/yagl4j&amp;utm_campaign=Badge_Grade)
 [![CircleCI](https://circleci.com/gh/delta-leonis/algieba.svg?style=svg)](https://circleci.com/gh/delta-leonis/algieba)
 
 You'll need at least Java 1.8 ([jre](https://www.java.com/download/)
 /[jdk](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html))
 to run `frosty`.
 
-## Documentation
+<!---## Dependency
```

Comment:
This shouldn't be commented out; nor is frosty deployed on maven currently

Reply:
i left it comment to discomment it when will be on maven, so how do u prefer it?

Reply:
Let's park this PR until frosty is deployed to maven.